src/Components/Modals/DeleteModal/DeleteModal.scss

```diff
@@ -0,0 +1,38 @@
+@import '../../../common.scss';
+
+.delete-modal{

```

Comment:
I would create modal window component that contains logic of only modal window without content. Moreover you could use react portals for modal window. Look at https://ru.reactjs.org/docs/portals.html